### PR TITLE
Fixing exception message

### PR DESCRIPTION
**Pull request by alexabuko**
This was an issue in a different pack (but the fix for that is within this one so I am pushing my changes here)

### CODE REVIEW

**Review by vtomole — APPROVED** [review]
LGTM. Thanks.

**Comment by vtomole** [comment]
Congrats on your first PR merge :partying_face: :champagne:

**Comment by singular-value** [comment]
Congrats!

File: applications_superstaq/superstaq_client.py [hunk 1]
```diff
@@ -285,12 +285,10 @@ def _handle_status_codes(self, response: requests.Response) -> None:
             )
 
         if response.status_code not in self.RETRIABLE_STATUS_CODES:
-            message = response.reason
-            if response.status_code == 400:
-                if "message" in response.json():
-                    message = response.json()["message"]
-                else:
-                    message = str(response.text)
+            if "message" in response.json():
```
[the remark this answers is not in the document]
Comment (vtomole) [reply applications_superstaq/superstaq_client.py:288]: Discussed offline: `200` is handled earlier in the code.

**Pull request merged**